packages/evm/src/memory.ts

```diff
@@ -63,20 +63,19 @@ export class Memory {
    * It fills up the difference between memory's length and `offset + size` with zeros.
    * @param offset - Starting position
    * @param size - How many bytes to read
+   * @param avoidCopy - Avoid memory copy if possible for performance reasons (optional)
    */
-  read(offset: number, size: number): Buffer {
+  read(offset: number, size: number, avoidCopy?: boolean): Buffer {
     this.extend(offset, size)
 
     const returnBuffer = Buffer.allocUnsafe(size)
```

Comment:
shouldn't this alloc go below avoidCopy return?

Reply:
Good point, will update.